pkgs/development/ocaml-modules/lwt-canceler/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildDunePackage, lwt }:
+
+buildDunePackage {
+  pname = "lwt-canceler";
+  version = "0.2";
+
+  src = builtins.fetchurl {
+    url = https://gitlab.com/nomadic-labs/lwt-canceler/-/archive/v0.2/lwt-canceler-v0.2.tar.gz;
```

Comment:
What about using `fetchFromGitLab`?

Reply:
Did not know it existed, these archives are what opam points to. But if you rather I use `fetchFromGitLab` I can change all my PRs to use that

Reply:
Yes, it is better to use fetchFromGitLab (or fetchzip); see the relevant documentation: https://nixos.org/manual/nixpkgs/stable/#chap-pkgs-fetchers

Reply:
I have now updated this PR and will continue with the others, thank you